README.md

```diff
@@ -1,19 +1,19 @@
-# bs-react-intl-ppx
+# melange-react-intl-ppx
+
+These are the [Melange](https://melange.re/) bindings for [react-intl](https://github.com/formatjs/formatjs/tree/main/packages/react-intl).
```

Comment:
Aren't the bindings here? https://github.com/cca-io/rescript-react-intl.

```suggestion
This is a [Melange](https://melange.re/) [preprocessor](https://ocaml.org/docs/metaprogramming) that eases the creation of messages when using [rescript-react-intl](https://github.com/cca-io/rescript-react-intl).
```

Reply:
Should the bindings and the ppx leave under the same repo, and probably the same package? cc @ixzzd

Reply:
Good catch. Fixed README:
https://github.com/ahrefs/bs-react-intl-ppx/blob/9e7418251f817c0a44d47820f923285702de54dd/README.md?plain=1#L3